lib/elixir/lib/registry.ex

```diff
@@ -653,6 +676,75 @@ defmodule Registry do
   end
 
   @doc """
+  Unregister entries for a given key matching a pattern.
+
+  ## Examples
+
+  For unique registries it's the same as unregister:
```

Comment:
Actually, that's not true, it can be used to remove something only if it has a certain value. :)